src/renderers/shared/stack/event/EventPluginRegistry.js

```diff
@@ -278,9 +278,12 @@ var EventPluginRegistry = {
       if (!dispatchConfig.phasedRegistrationNames.hasOwnProperty(phase)) {
         continue;
       }
-      var pluginModule = EventPluginRegistry.registrationNameModules[
-        dispatchConfig.phasedRegistrationNames[phase]
-      ];
+      let pluginModule;
+      if (dispatchConfig.phasedRegistrationNames) {
+        pluginModule = EventPluginRegistry.registrationNameModules[
+          dispatchConfig.phasedRegistrationNames[phase]
+        ];
+      }
       if (pluginModule) {
```

Comment:
I would put this `if` inside of the previous one. This way you don't need to make `let pluginModule;` outside of the block